person.rb

```diff
@@ -1,21 +1,25 @@
 require './nameable'
 
 class Person < Nameable
+  attr_accessor :name, :age, :rentals
+  attr_reader :id
+
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..100)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
 
   def correct_name
     @name
   end
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235165804-0f8234b5-a8b0-44dd-83da-5224ff3c84b2.png)

**[OPTIONAL]**

- [ ] It would be nice if you fix all the highlighted links in your Table of contents to make your readme descriptive and professional 👍